backend/src/polydash/polygon/top_peers.py

```diff
@@ -0,0 +1,60 @@
+from fastapi import APIRouter
+from pony.orm import desc, db_session
+
+from polydash.miners_ratings.model import MinerRisk
+from polydash.polygon.deanon.model import DeanonNodeByBlock, DeanonNodeByTx, PeerToIP
+
+TOP_NODES_LIST_SIZE = 10
```

Comment:
Maybe it's better to have this as an endpoint argument to make it more flexible. The default can be left `10`.

Reply:
Ok, makes sense